static/app/components/profiling/FlamegraphTooltip/flamegraphTooltip.tsx

```diff
@@ -22,6 +23,19 @@ export function formatWeightToProfileDuration(
   return `(${Math.round((frame.totalWeight / flamegraph.profile.duration) * 100)}%)`;
 }
 
+function formatFileNameAndLineColumn(frame: FlamegraphFrame): string | null {
+  if (!frame.frame.file) {
+    return null;
+  }
+  if (frame.frame.line && frame.frame.column) {
```

Comment:
can we have line 0 and col 0? I'm pretty sure no since we start counting from 1 there but just double checking

Reply:
I dont think so, but it makes sense to check for number anyways